src/components/MealType/MealType.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import img from './img/wtgain.png';
+
+const MealType = ({ title, imgUrl }) => {
+  return (
+    <div
+      className="h-64 w-screen md:h-screen md:w-1/2 flex items-center justify-center"
```

Comment:
can we make it responsive? 
so I was thinking that in smaller screen the width will be full and hight will be like 50vh